envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -47,15 +48,29 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
      * It contains the versions of XDS resources and the contents for a provided node JSON
      * extracted from Envoy's config_dump endpoint.
      */
-    @PostMapping("/snapshot")
+    @PostMapping("/snapshotV2")
```

Comment:
Maybe we should map it into both `/snapshotV2` and `/snapshot` so that one of our tools keeps working.

Reply:
How about handling it with a content type? `appliation/json` and `application/v2+json` handled by endpoint with `io.envoyproxy.envoy.api.v2.core.Node` and `application/v3+json` handled by endpoint taking `io.envoyproxy.envoy.config.core.v3.Node`. Snapshot endpoint requires and serves the same data but input format differs depending on version.

Reply:
Done